Wootric/WootricSDK-Android
# Fix layout & localization issues

- Move Edit Score, Dismiss & Send buttons up on Feedback view to prevent being covered by keyboard.
- Fix score view layout on tablets
- Remove last CtA button on tablets if custom Thank You message not set
- Show buttons in last view if custom thankyou or social links are set
- Add CHANGELOG.md & LICENSE.md files to repo
- Update gitignore file to ignore *.iml files, which are IDE files
- Fix http response decoding on WootricRemoteRequestTask.java to show correct characters unicode characters
- Add log to requests and responses on WootricRemoteRequestTask.java
- Add Edit Score to LocalizedTexts
- Fix Edit Score button to show localized text
- Fix Submit text on tablets to show localized text
- Add getters for social share question and social share decline on Settings.java
- Fix final thankyou message. If custom message is set, show it. If not, show default thankyou message from localized texts
- Fix final thankyou message on tablets. If custom message is set, show it. If not, show default thankyou message from localized texts
- Use default hint color for feedback placeholder on wootric_survey_layout.xml
- Set buttons textSize from dimens files on wootric_survey_layout.xml
- Add dimens.xml to change font size for different screen densities
- Update settings test
- Remove .iml files

To test:
- First try without setting a WootricCustomThankYou. If you can try with different language better
- Then set a WootricCustomThankYou with just

```java
customThankYou.setText("Thank you");
customThankYou.setDetractorText("Detractor thank you");
customThankYou.setPassiveText("Passive thank you");
customThankYou.setPromoterText("Promoter thank you");
```
to verify that texts on the last view changes correctly

Then also set links to see the last CtA buttons

```java
customThankYou.setLinkText("Thank you link text");
customThankYou.setDetractorLinkText("Detractor link text");
customThankYou.setPassiveLinkText("Passive link text");
customThankYou.setPromoterLinkText("Promoter link text");
customThankYou.setLinkUri(Uri.parse("http://wootric.com/thank_you"));
customThankYou.setDetractorLinkUri(Uri.parse("http://wootric.com/detractor_thank_you"));
customThankYou.setPassiveLinkUri(Uri.parse("http://wootric.com/passive_thank_you"));
customThankYou.setPromoterLinkUri(Uri.parse("http://wootric.com/promoter_thank_you"));
customThankYou.setScoreInUrl(true);
customThankYou.setCommentInUrl(true);
```
